frontend/src/components/pages/RegistrantExperience/RegistrationSteps/AdditionalInfo/QuestionCards/EarlyDropOffLatePickupCard.tsx

```diff
@@ -32,10 +33,18 @@ const EarlyDropOffLatePickupCard = ({
   return (
     <QuestionsCardWrapper title="Early Drop-off and Late Pick-up">
       <Wrap>
-        <WrapItem px="40px" py="12px">
-          <FormControl isRequired isInvalid={invalid}>
-            <FormLabel fontWeight="bold" fontSize="18px">
-              Do your camper(s) require early drop-off or late pick-up?
+        <WrapItem px={{ sm: "5", lg: "20" }} py={4}>
+          <FormControl isInvalid={invalid}>
+            <FormLabel textStyle={{ sm: "xSmallBold", lg: "buttonSemiBold" }}>
+              Do your camper(s) require early drop-off or late pick-up?{" "}
+              <Text
+                as="span"
+                color="text.critical.100"
+                fontSize="xs"
+                verticalAlign="super"
+              >
+                <RequiredAsterisk />
```

Comment:
We had the `isRequired` prop before, but this wasn't being rendered properly- the asterisk was being rendered on the next line and I honestly couldn't find a way to fix it since it's a chakra base component.
Adding our own `<RequiredAsterisk/>` component is what we did in the `CamperCard.tsx` component too, so I just used the same approach for this.
![image](https://user-images.githubusercontent.com/30396273/212401160-1528539b-a435-4eff-b8f8-79f0c661dc76.png)